test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
### Your test.md file
![test (copy)](https://user-images.githubusercontent.com/93039359/184937652-4fde2c63-278e-4476-b2a2-a6287f707a2a.png)

- [ ] You have added your test.md file in .gitignore but it is still in your Github Repository. Please fix this issue why your .gitignore is not working :+1:

Reply:
- [ ] This issue has been resolved :heavy_check_mark: